### PR TITLE
[xml] russian.xml  v8.7.1

### PR DESCRIPTION
**Pull request by GOLEM777**

### CODE REVIEW

**Comment by donho** [comment]
@GOLEM777 
Could you resolve the conflict please?

File: PowerEditor/installer/nativeLang/russian.xml [hunk 1]
```diff
@@ -672,6 +672,7 @@ Updated to v8.7.0:
 					<Item id="2231" name="Общий курсив шрифтов"/>
 					<Item id="2232" name="Общее подчёркивание шрифтов"/>
 					<Item id="2234" name="Перейти к настройкам"/>
+					<Item id="2235" name="Что значит Глобальное Замещение?"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/russian.xml:675]: "Global override" should not be translated.

**Review by donho — CHANGES_REQUESTED** [review]
1. Restore "Global override"
2. Solve the conflict

**Comment by GOLEM777** [comment]
Problem in block 
```
<<<<<<< patch-45
					<ComboBox id="6307">
						<Element name="Бездействие"/>
						<Element name="Свернуть в"/>
						<Element name="Закрыть в"/>
					</ComboBox>
					<Item id="6308" name="системный трей"/>
=======
>>>>>>> master
```

**Comment by GOLEM777** [comment]
Patch queue conflict.
Need to recreate. This has happened before.
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/cd45afc0208f92c21afd286ce3f691f6f32d1989

**Comment by roysten69** [comment]
Designer (26)

**Pull request closed without merging**